CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 **Features**:
 
 - Session metrics extraction: Count crashed+abnormal towards errored_preaggr. ([#1274](https://github.com/getsentry/relay/pull/1274))
+- Implement AWS extension as an actor. ([#1277](https://github.com/getsentry/relay/pull/1277))
```

Comment:
Let's make this a slightly more descriptive changelog entry around what the feature enables.

Reply:
this